packages/backend/src/peripherals/database/migrations/023_tx_count.ts

```diff
@@ -0,0 +1,27 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.createTable('tx_count', function (table) {
+    table.string('project_id').notNullable()
+    table.dateTime('unix_timestamp').notNullable()
+    table.integer('count').notNullable()
+    table.integer('block_number')
```

Comment:
This table has no primary key and has no indices. I think that it would be good to have a shared primary key of `project_id` and `block_number`. Also, why is `block_number` not `notNullable()`?

Reply:
So the idea was primarily that some of the projects don't give us block_number (ex. all the StarkEx implementations)

Reply:
Thinking of this rn, we should probably have 2 tables, where 1 is only for raw block data and the second would hold aggregated data, ie for StarkEx